RUFAS/routines/field/crop/heat_units.py

```diff
@@ -42,52 +69,83 @@ def absorb_heat_units(self, mean_air_temperature: float = None,
+               the middle of the crop temperature window.
+            4. If the crop's temperature range is entirely within the air temperature range, accumulation equals
+               the middle of the air temperature range.
+
         """
         self.assign_new_heat_units(air_temperature)
         self.add_heat_units()
 
     def assign_new_heat_units(self, air_temperature: float = None) -> None:
-        """assign new heat units based on if the alternative accumulation method is to be used"""
+        """
+        Assign new heat units based on whether the alternative accumulation method is to be used.
+
+        Parameters
+        ----------
+        air_temperature : Optional[float], optional
+            The average air temperature during the day (°C).
+
+        """
         if self.data.use_heat_unit_temperature or (air_temperature is None):  # alternative method
             self.data.new_heat_units = self._determine_new_heat_units(self.data.heat_unit_temperature,
                                                                       self.data.minimum_temperature)
         else:  # main method
             self.data.new_heat_units = self._determine_new_heat_units(air_temperature, self.data.minimum_temperature)
 
     def add_heat_units(self) -> None:
-        """add newly acquired heat units to accumulated heat units"""
+        """
+        Add newly acquired heat units to accumulated heat units.
+        """
         self.data.accumulated_heat_units += self.data.new_heat_units
 
     # TODO: add these warnings to output manager at a later date.
     def _check_absorb_heat_for_input_errors(self, mean_air_temperature: float = None,
                                             min_air_temperature: float = None,
-                                            max_air_temperature: float = None):
-        """raises errors if inputs given for absorb_heat_units don't make sense with the value of the
-        use_heat_unit_temperature attribute"""
+                                            max_air_temperature: float = None) -> None:
+        """
+        Raises errors if inputs given for absorb_heat_units don't make sense with the value of the
+        use_heat_unit_temperature attribute.
+
+        Parameters
+        ----------
+        mean_air_temperature : Optional[float], optional
+            Average air temperature for the day (°C).
+        min_air_temperature : Optional[float], optional
+            Minimum air temperature for the day (°C).
+        max_air_temperature : Optional[float], optional
+            Maximum air temperature for the day (°C).
+
+        Raises
+        ------
+        ValueError
+            If `use_heat_unit_temperature` is True, both `min_air_temperature` and `max_air_temperature` must be
+            provided.
+        ValueError
+            If `use_heat_unit_temperature` is False, `mean_air_temperature` must be provided.
```

Comment:
This does not follow the style of "Raises" sections in #1074.
```suggestion
        ValueError
            If `use_heat_unit_temperature` is True and both `min_air_temperature` and `max_air_temperature` are not provided.
            If `use_heat_unit_temperature` is False and `mean_air_temperature` is not provided.
```